helm/azuredisk-csi-driver-app/values.yaml

```diff
@@ -52,8 +52,10 @@ controller:
   livenessProbe:
     healthPort: 29602
   runOnMaster: false
+  masterNodeSelector:
+    "kubernetes.io/role": master
```

Comment:
I'm not 100% sure this will work on capi. But nevermind we can always fix it afterwards

Reply:
yeah in CAPI i need to change that and the azurefile credential location, that's why i moved both to the values but left the default to match what is currently used in vintage 

https://github.com/giantswarm/default-apps-azure/pull/7/commits/fd73d68203f33ccf614c536a14a222d76f010a2b#diff-9cfdd527b22ba84cdc4ad1df7c0e647c13dea797bc547e867476568f374778fd

Reply:
Gr8 thx for explaining

Reply:
that's why its value that can be overridden in `default-apps-azure` :)